contentgrid-spring-data-rest/src/testFixtures/java/com/contentgrid/spring/test/fixture/invoicing/model/PromotionCampaign.java

```diff
@@ -32,13 +32,12 @@ public class PromotionCampaign {
 
     @Column(updatable = false, nullable = false)
     @CollectionFilterParam(value = "promo_code")
-    @RestResource(rel = "d:promo-code")
```

Comment:
Note: this was just a plain incorrect annotation that I cleaned up: promoCode is not a relation, so can't be a `@RestResource`